msvc/ut-build.props

```diff
@@ -4,7 +4,7 @@
   <ItemDefinitionGroup>
     <ClangCompile>
       <AdditionalOptions>-Werror=microsoft %(AdditionalOptions)</AdditionalOptions>
-      <PreprocessorDefinitions>_WINOBJC_DO_NOT_USE_NSLOG=;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_WINOBJC_DO_NOT_USE_NSLOG=;_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
_CRT_SECURE_NO_WARNINGS: gasp.  we shouldn't have *that* many warnings we want to suppress?